apps/dashboard/config/configuration_singleton.rb

```diff
@@ -39,7 +39,8 @@ def boolean_configs
       :csp_enabled          => false,
       :csp_report_only      => false,
       :bc_dynamic_js        => false,
-      :per_cluster_dataroot => false
+      :per_cluster_dataroot => false,
+      :jobs_app_alpha       => true
```

Comment:
I think tests will pass if we flip this default. Seems like that's what we want anyhow - we don't want to enable this for everyone just yet.